MAINTAINERS.md

```diff
@@ -27,14 +27,14 @@ To release a new minor version, you can use the `npm run releases:guides:minor`
 The script will create and update the necessary files for you, showing you a URL at the end that you can use to create the PR.
```

Comment:
I think the line above here needs to be singularized 
`npm run releases:guides:minor` --> `npm run release:guides:minor`
Since this line wasn't changed I can't seem to get Github to let me make a suggestion directly inline.